### PR TITLE
fix(rolling-upgrade): enable use_prepared_loaders

### PR DESCRIPTION
**Pull request by fruch**
Since we were seen issue with the docker based loader,
and unclear performence and timing effect on the rolling
upgrade test, we should be using the fallback to VMs now

Closes: https://github.com/scylladb/scylla-cluster-tests/issues/5526

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -468,6 +469,18 @@ def system_log(self):
...
 
+    @property
+    def cassandra_stress_version(self):
+        if not self._cassandra_stress_version:
+            result = self.remoter.run(cmd="cassandra-stress version", ignore_status=True, verbose=True)
+            match = re.search("Version: (.*)", result.stdout)
+            if match:
+                self._cassandra_stress_version = match.group(1)
+            else:
+                self.log.error("C-S version not found!")
```
Comment (soyacz) [inline sdcm/cluster.py:480]: add `result.stdout` to error message for quicker issue investigation.

File: sdcm/cluster.py [hunk 2]
```diff
@@ -4812,6 +4875,36 @@ def _parse_cs_summary(lines):
             return {}
         return results
 
+    @staticmethod
+    def _parse_cs_results(lines):
```
Comment (soyacz) [inline sdcm/cluster.py:4879]: where is it used?

Comment (fruch) [reply sdcm/cluster.py:4879]: it's not, it a revert of a commit which I've deleted it. (the revert if cause of different reason)

File: sdcm/cluster.py (hunk 1 above)
Comment (fruch) [reply sdcm/cluster.py:480]: I need to remove this, it's not actually being used, it's back cause it's part of a revert I'm doing in this PR

the revert is to return the code that install c-s on-top of the loaders, (it was remove as part of switching to docker based loaders)

File: sdcm/cluster.py (hunk 2 above)
Comment (fruch) [reply sdcm/cluster.py]: I'll remove it from the revert

File: sdcm/cluster.py (hunk 1 above)
Comment (fruch) [reply sdcm/cluster.py]: I'll remove it from the revert

**Review by roydahan — APPROVED** [review]

**Pull request merged**